Add documentation about using custom OpenAI models (#46)
wagtail/wagtail-ai: docs/ai-backends.md

    }
}
```

#### Using a custom OpenAI or OpenAI-compatible model

llm supports adding custom OpenAI models. This may be necessary if:

> - You want to use a model that's not supported by llm package yet.

Author: I do wonder if we want to monospace the library name. We don't write `Wagtail`, `Windows`, `Django` etc.

> - You want to use a model that's not supported by llm package yet.

Reviewer: I don't mind either way, as long as we are consistent throughout

    }
}
```

#### Using a custom OpenAI or OpenAI-compatible model

The "LLM" library supports adding custom OpenAI models. This may be necessary if:

- You want to use a model that's not supported by the "LLM" library yet.
- You want to use a proxy for OpenAI requests.

You can find the "LLM" library specific instructions at: https://llm.datasette.io/en/stable/other-models.html#adding-more-openai-models.

> 1. Find the "LLM" library's directory. You can set a custom one with the. You can
>    set a custom one with the

Reviewer: ```suggestion
1. Find the "LLM" library's directory. You can set a custom one with the
```

   [`LLM_USER_PATH`](https://llm.datasette.io/en/stable/setup.html#setting-a-custom-directory-location)

>    setting. You can then confirm the path with the following shell command:
>    `dirname "$(llm logs path)"`.

Reviewer: ```suggestion
   setting. To confirm the path, you can use the following shell command:
   `dirname "$(llm logs path)"`.
```

2. Create `extra-openai-models.yaml` as noted in
   [the "LLM" library's documentation](https://llm.datasette.io/en/stable/other-models.html#adding-more-openai-models).
   For example to set up a proxy:
   ```yaml
    - model_id: customgateway-gpt-3.5-turbo
      model_name: gpt-3.5-turbo
      api_base: "https://yourcustomproxy.example.com/"
      headers:
        apikey: your-api-key
   ```
3. Set the `MODEL_ID` in the Wagtail AI settings in your Django project

>    settings to the `model_id` you set in the yaml file.

Reviewer: ```suggestion
   settings file to the `model_id` you added in `extra-openai-models.yaml`.
```
